### PR TITLE
Add 1.22 Release Shadows and remove 1.21

### PR DESCRIPTION
**Pull request by guineveresaenger**
/cc @savitharaghunathan 
/sig-release

### CODE REVIEW

File: groups/sig-release/groups.yaml [hunk 1]
```diff
@@ -412,26 +412,28 @@ groups:
...
+      - jgavinray@linux.com # 1.22 Bug Triage Shadow
+      - kunalkushwaha453@gmail.com # 1.22 Comms Shadow
+      - rajula96reddy@gmail.com # 1.22 Comms Shadow
+      - butlerjl@amazon.com # 1.22 Comms Shadow
+      - klkfr@amazon.com # 1.22 Comms Shadow
+      - joseph.r.sandoval@gmail.com # 1.22 Enhancements Shadow
+      - xander@grzy.dev # 1.22 Enhancements Shadow
+      - supriyapremkumar1@gmail.com # 1.22 Enhancements Shadow
+      - nng.grace@gmail.com # 1.22 Enhancements Shadow
+
```
Comment (savitharaghunathan) [inline groups/sig-release/groups.yaml:439]: Thanks @guineveresaenger :)
Enhancements shadow list is missing Rey Lejano. Can you add his email as well?

Comment (savitharaghunathan) [reply groups/sig-release/groups.yaml:439]: Ping @guineveresaenger

**Review by spiffxp — APPROVED** [review]
/approve 
/hold
If you'd rather merge now and fix in a followup you can remove hold

**Comment by savitharaghunathan** [comment]
> If you'd rather merge now and fix in a followup you can remove hold

Thanks, @spiffxp :) I am going to remove the hold. I have a follow-up PR coming in today and I can tackle this one in that.

/hold cancel

**Pull request merged**